### ISSUE TITLE
Missing references to secnodary color

### ISSUE THREAD

**Issue by mariocalin** [stisla/stisla#108]
Classes _badge-secondary_ and _card-secondary_ don't apply the **secondary** color set in __variables.scss_ files. 

```scss
/* _badge.scss file */
  &.badge-secondary {
    background-color: color(fontdark);
  }

/* _card-scss file */
.card-secondary {
    border-top: 2px solid color(fontdark);
  }
```

Shouldn't they reference to the _secondary_ instead of _fontdark_?

**Comment by nauvalazhar** [comment]
Hm, can you try it and give me a screenshot, please?

**Comment by mariocalin** [comment]
Line 152 file -> 
https://github.com/stisla/stisla/blob/master/sources/scss/override/_card.scss

And line 20 file -> 
https://github.com/stisla/stisla/blob/master/sources/scss/override/_badge.scss

They are both using the fontdark so, wen I change the secondary color in the __variable.scss_ file and redbuild the template, I expect that the .secondary class in badge and card will use the secondary color and not the fontdark color.

I have modified those lines and reference the secondary color in local development and it generates correctly the _-secondary_ class.

Screenshots:
In my case I changed the secondary color to this kind of pink (# f90d5c). 

Cards
![screenshot](https://i.postimg.cc/tgGFnbfT/screenshot-localhost-8081-2019-12-11-15-31-43.png)

Badges
![screenshot2](https://i.postimg.cc/vTnVZSz9/screenshot-localhost-8081-2019-12-11-15-32-53.png)

**Comment by nauvalazhar** [comment]
Oh, I think you're right. Thank you for your correction. Can you make a pull request, please?

**Comment by mariocalin** [comment]
Issue solved in #109
